lxc/config_trust.go

```diff
@@ -93,6 +100,11 @@ func (c *cmdConfigTrustAdd) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	// Validate flags.
+	if !shared.StringInSlice(c.flagType, []string{"client", "metrics"}) {
+		return fmt.Errorf("Unknown certificate type %q", c.flagType)
+	}
+
```

Comment:
We should add a check that if `metrics` is used that we do have the metrics API extension.